remote_monitoring_demo/update_results.json

```diff
@@ -34,16 +32,16 @@
          "results": {
            "general_info": {
              "max": 0.84,
-             "mean": 0.12,
+             "avg": 0.12,
```

Comment:
cpuUsage needs to have `min`, `max`, `avg` and `sum`.
cpuThrottle needs to have `max`, `avg` and `sum`.